Automattic/newspack-plugin
# Initial onboarding overhaul

### All Submissions:






### Changes proposed in this Pull Request:



First step of a larger onboarding overhaul. 
Closes #806 

Also, this PR proposes using [Tachyons CSS framework](http://tachyons.io/) for easier UI building. Note that this framework is not meant for building components or being used _in_ `newspack-components`, since it's not shipped with the versions distributed on npm. This framework is meant just as a lightweight tool for faster UI building.

### How to test the changes in this Pull Request:

1. Install the plugin on a fresh WP site 
2. Observe a much simplified onboarding experience:
3. All required software (plugins, themes), as well as starter content, is installed in the background, with only a progress bar as UI feedback
4. If any step of this fails, a warning will be displayed. User has to reload the page to resume.
5. The profile information is stripped to just country & currency
6. Jetpack, Site Kit, and Mailchimp connections are all on one view
7. After choosing the theme the setup is complete. 

### Other information:
